zerver/data_import/slack.py

```diff
@@ -1120,14 +1120,29 @@ def build_reactions(
     for realm_emoji in zerver_realmemoji:
         realmemoji[realm_emoji["name"]] = realm_emoji["id"]
 
+    # Slack's data exports use encode skin tone variants on emoji
+    # reactions like this: `clap::skin-tone-2`. For now, we only
+    # use the name of the base emoji, since Zulip's emoji
+    # reactions system doesn't yet support skin tone modifiers.
+    # We need to merge and dedup reactions, as someone may have
+    # reacted to `clap::skin-tone-1` and `clap::skin-tone-2`, etc.
+    mergedreactions: Dict[str, Any] = {}
```

Comment:
Worth mentioning this is what it is because mypy linter balked on several simpler definitions.

Reply:
Yeah, I think defining a TypedDict is what you'd want if you want if one wanted to type this; I'm not convinced that's worth doing given how locally this is used.